### ISSUE TITLE
The whole Jacobian Matrix for AGV

### ISSUE THREAD

**Issue by Harrydsk** [petercorke/robotics-toolbox-python#456]
Can I see the whole Jacobian Matrix of AGV. This is my defination of it. It seems that its differrent with what the aritical shows.
![Snipaste_2024-09-23_11-46-58](https://github.com/user-attachments/assets/561814c2-158e-46d6-8257-7708d577ad58)
![图片1](https://github.com/user-attachments/assets/0bd9557d-c1e0-4764-a3de-257560ad733a)

**Comment by petercorke** [comment]
Yes -- see `VehicleBase.Fx()`/`Fv()` in `roboticstoolbox/mobile/Vehicle.py` (and the concrete `Bicycle`/`Unicycle`/`DiffSteer` subclasses) for the full Jacobian expressions.
